### PR TITLE
Fix map implementation in GraphEdge class.

### PR DESCRIPTION
**Pull request by qpautrat**
Aims to fix https://github.com/facebook/php-graph-sdk/issues/714

### CODE REVIEW

File: src/Facebook/GraphNodes/GraphEdge.php [hunk 1]
```diff
@@ -235,4 +235,18 @@ public function getTotalCount()
 
         return null;
     }
+
+    /**
+     * @{inherited}
```
Comment (yguedidi) [inline src/Facebook/GraphNodes/GraphEdge.php]: the "standard" one is `@inheritDoc`

File: tests/GraphNodes/GraphEdgeTest.php [hunk 2]
```diff
@@ -96,4 +97,28 @@ public function testCanInstantiateNewPaginationRequest()
         $this->assertEquals('/v1337/998899/photos?access_token=foo_token&after=foo_after_cursor&appsecret_proof=857d5f035a894f16b4180f19966e055cdeab92d4d53017b13dccd6d43b6497af&foo=bar&limit=25&pretty=0', $nextPage->getUrl());
         $this->assertEquals('/v1337/998899/photos?access_token=foo_token&appsecret_proof=857d5f035a894f16b4180f19966e055cdeab92d4d53017b13dccd6d43b6497af&before=foo_before_cursor&foo=bar&limit=25&pretty=0', $prevPage->getUrl());
     }
+
+    public function testCanMapOverNodes()
+    {
+        $graphEdge = new GraphEdge(
+            $this->request,
+            [new GraphNode(['name' => 'dummy'])],
```
Comment (yguedidi) [inline tests/GraphNodes/GraphEdgeTest.php]: I'd add another node to test that it's loop

**Comment by SammyK** [comment]
Thanks so much @qpautrat! @yguedidi This looks good to me, should I merge? :)

**Comment by qpautrat** [comment]
Done !

**Comment by SammyK** [comment]
Thanks @qpautrat! :)

**Pull request merged**